polychrom/starting_conformations.py

```diff
@@ -119,7 +119,7 @@ def create_random_walk(step_size, N):
 
 
 def create_constrained_random_walk(
-    N, constraint_f, starting_point=(0, 0, 0), step_size=1.0
+    N, constraint_f, starting_point=(0, 0, 0), step_size=1.0, polar_fixed=None
 ):
     """
     Creates a constrained freely joined chain of length N with step step_size.
```

Comment:
It looks good to me! Would be very nice to add type hints here, to make it easier for the user to understand. For constraint_f it would be good to add Callable https://docs.python.org/3/library/typing.html#typing.Callable